jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/JRubyPlugin.groovy

```diff
@@ -62,14 +64,12 @@ class JRubyPlugin implements Plugin<Project> {
             project.logger.debug 'Adding rubygems(String?) method to project RepositoryHandler'
             project.repositories.metaClass.rubygems << { String repoUrl = null ->
                 repoUrl = repoUrl ?: RUBYGEMS_ORG_URL
-                // can not cast
-                Object embedded = embeddedServer(project)
-                String path = embedded.addRepository(repoUrl)
+                setupMavenGemProtocol(project)
```

Comment:
Is this idempotent and acceptable to call multiple times? I imagine it won't be common, but we should support:

```gradle
repositories {
  rubygems('https://rubygems.org')
  rubygems('https://myartifactory/api/gems/some-repo')
```

Would there be a downside to calling setupMavenGemProtocol when the project is initially loaded?